Style.css

```diff
@@ -0,0 +1,1029 @@
+  text-decoration: none;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.pro-art {
+  padding-top: 14%;
+  margin-top: 4%;
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 99.94%
+    ),
+    url("images/img_2.png");
+}
+
+.headline-4 {
+  width: 295px;
+  height: 81px;
+  margin-left: 4%;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+
+  /* or 138% */
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text-content-2 {
+  width: 301px;
+  height: 99px;
+  margin-left: 4%;
+
+  /* Theme 3 / Body Reguler 2 */
+
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* or 150% */
+  margin-top: -3%;
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.btn-html-1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 67px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.btn-html-1 a {
+  width: 43px;
+  height: 20px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+
+  /* identical to box height, or 133% */
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.btn-bootstrap-1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 115px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.btn-bootstrap-1 a {
+  width: 71px;
+  height: 20px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+
+  /* identical to box height, or 133% */
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.btn-ruby-1 {
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 71px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.btn-ruby-1 a {
+  width: 37px;
+  height: 20px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+
+  /* identical to box height, or 133% */
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.btn-sub-2 {
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  border: none;
+  background: #ff6b00;
+}
```

Comment:
- [ ] Kindly implement the button interactions (enable, hover, etc.) to match the given design.
- Here is the given instruction related to button interactions.
- ![image](https://github.com/MohammadYaser/Portfolio/assets/60083437/adbfb0dd-f104-49f8-a7ca-c703ef3c615f)